R/pkg/R/sparkR.R

```diff
@@ -93,7 +93,7 @@ sparkR.stop <- function() {
 #' sc <- sparkR.init("local[2]", "SparkR", "/home/spark",
 #'                  list(spark.executor.memory="1g"))
 #' sc <- sparkR.init("yarn-client", "SparkR", "/home/spark",
-#'                  list(spark.executor.memory="1g"),
+#'                  list(spark.executor.memory="4g", spark.driver.memory="2g"),
```

Comment:
updated. created JIRA for additional programming guide change: https://issues.apache.org/jira/browse/SPARK-11407

I could take a shot at the doc change if you'd like